packages/synapse-client/src/SynapseClient.ts

```diff
@@ -0,0 +1,164 @@
+import {
+  AccessApprovalServicesApi,
+  AccessRequirementServicesApi,
+  ActivityServicesApi,
+  AsynchronousJobServicesApi,
+  AuthenticationServices2FAApi,
+  AuthenticationServicesApi,
+  BaseAPI,
+  CertifiedUserServicesApi,
+  ChallengeServicesApi,
+  DataAccessServicesApi,
+  DefaultConfig,
+  DiscussionServicesApi,
+  DockerAuthorizationServicesApi,
+  DockerCommitServicesApi,
+  DockerRegistryEventServicesApi,
+  DOIServicesApi,
+  DownloadListServicesApi,
+  DrsServicesApi,
+  EntityBundleServicesV2Api,
+  EntityServicesApi,
+  EvaluationServicesApi,
+  FileServicesApi,
+  FormServicesApi,
+  JSONSchemaServicesApi,
+  LogServiceApi,
+  MembershipInvitationServicesApi,
+  MembershipRequestServicesApi,
+  MessageServicesApi,
+  OpenIDConnectServicesApi,
+  PrincipalServicesApi,
+  ProjectSettingsServicesApi,
+  RecycleBinServicesApi,
+  SearchServicesApi,
+  StatisticsServicesApi,
+  StorageReportServicesApi,
+  SubscriptionServicesApi,
+  TableServicesApi,
+  TeamServicesApi,
+  User38GroupServicesApi,
+  UserProfileServicesApi,
+  VerificationServicesApi,
+  WikiPageServices2Api,
+  WikiPageServicesApi,
+} from './generated'
+
+/**
+ * Creates one class that encapsulates all sets of Synapse API services.
+ */
+export class SynapseClient extends BaseAPI {
```

Comment:
The generator creates one API client class per Java controller. I created this convenience wrapper over every service.